sql/sys_vars.cc

```diff
@@ -391,9 +391,14 @@ static Sys_var_mybool Sys_vers_force(
        "vers_force", "Force system versioning for all created tables",
        GLOBAL_VAR(vers_force), CMD_LINE(OPT_ARG), DEFAULT(FALSE));
 
-static Sys_var_mybool Sys_vers_hide(
-       "vers_hide", "Hide system versioning from being displayed in table info",
-       GLOBAL_VAR(vers_hide), CMD_LINE(OPT_ARG), DEFAULT(FALSE));
+static const char *vers_hide_keywords[]= {"AUTO", "IMPLICIT", "FULL", "NEVER", NullS};
```

Comment:
I guess `NullS` presents here for consistency. But why being consistent with a strange ancient MySQL stuff? Everybody knows about `NULL`. Using `NullS` makes code harder to read.